pkg/operator/tpr.go

```diff
@@ -59,7 +59,7 @@ func (o *Operator) createTPR() error {
 		}
 	}
 
-	return err
```

Comment:
@travisn line 58 right? I see it but i still think we should return an error because the TPR was not created and that will manifest later in the flow. But if we are going to swallow the errors that are not alreadyExist, we should at least log them in my opinion